walkthroughs/howto-k8s-ingress-gateway/v1beta2/manifest.yaml.template

```diff
@@ -0,0 +1,621 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${APP_NAMESPACE}
+  labels:
+    mesh: ${MESH_NAME}
+    gateway: ingress-gw
+    appmesh.k8s.aws/sidecarInjectorWebhook: enabled
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: Mesh
+metadata:
+  name: ${MESH_NAME}
+spec:
+  namespaceSelector:
+    matchLabels:
+      mesh: ${MESH_NAME}
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualGateway
+metadata:
+  name: ingress-gw
+  namespace: ${APP_NAMESPACE}
+spec:
+  namespaceSelector:
+    matchLabels:
+      gateway: ingress-gw
+  podSelector:
+    matchLabels:
+      app: ingress-gw
+  listeners:
+    - portMapping:
+        port: 8088
+        protocol: http
+      healthCheck:
+        path: /ping
+        healthyThreshold: 6
+        intervalMillis: 6000
+        protocol: http
+        timeoutMillis: 3000
+        unhealthyThreshold: 3
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: GatewayRoute
+metadata:
+  name: gateway-route-headers
+  namespace: ${APP_NAMESPACE}
+spec:
+  httpRoute:
+    match:
+      prefix: "/headers"
+    action:
+      target:
+        virtualService:
+          virtualServiceRef:
+            name: color-headers
```

Comment:
How is this associated with the gateway created above?

Reply:
Notice the nameSelector in virtual gateway `gateway: ingress-gw`. GatewayRoute is associated with the VirtualGateway that [has the nameSelector](https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/pkg/virtualgateway/membership_designator.go#L82) for the namespace in scope

Reply:
Thanks for the clarification -- forgot we had restricted this to 1 per namespace.